plugins/python-build/share/python-build/anaconda3-2020.11

```diff
@@ -0,0 +1,19 @@
+case "$(anaconda_architecture 2>/dev/null || true)" in
+"Linux-ppc64le" )
+  install_script "Anaconda3-2020.11-Linux-ppc64le" "https://repo.continuum.io/archive/Anaconda3-2020.11-Linux-ppc64le.sh#870535ada0a8ae75eeda8cd2bf7dde853ac9f4949b20e1b5641f1843a655f3b8" "anaconda" verify_py38
```

Comment:
Nice catch switching these to `verify_py38`. According to the release notes 3.8 was used by default in 2020.07, I'll file a separate PR to update that.